code/bmpman/bmpman.cpp

```diff
@@ -1892,6 +1892,7 @@ int bm_load_sub_slow(const char *real_filename, const int num_ext, const char **
 	}
 
 	// umm, that's not good...
+	Warning(LOCATION, "Could not open file %s!", res.full_name.c_str());
```

Comment:
Instead of displaying a Warning, I recommend outputting a message to the log.  I have a feeling that this would cause a lot of annoying Warnings for many mods.

Reply:
This only happens if a file exists but can't be opened. I think this should never happen. Especially since the file would be ignored in this case (a return value of `-1` bascially means the file doesn't exist).

Reply:
Ok, I stand corrected.